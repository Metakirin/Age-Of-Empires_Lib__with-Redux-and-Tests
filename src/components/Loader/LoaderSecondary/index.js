import loading from './Loading.gif'

const LoaderSecondary = () => {
  return (
    <div className="small_loader">
      <img src={loading} alt="loading..." style={{ width: '300px' }} />
    </div>
  )
}

export default LoaderSecondary
