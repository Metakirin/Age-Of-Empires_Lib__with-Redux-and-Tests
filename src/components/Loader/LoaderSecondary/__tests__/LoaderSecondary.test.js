import { configure, shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

import LoaderSecondary from '..';

configure({ adapter: new Adapter() });

describe('Should render correctly "LoaderSecondary"', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<LoaderSecondary />);
    });
    it('to Match Snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
    it('component is defined', () => {
        expect(wrapper).toHaveLength(1);
    });
    it('component have children div with "small_loader" class', () => {
        expect(wrapper.find('.small_loader')).toHaveLength(1);
    });
    it('component have children img', () => {
        expect(wrapper.find('img')).toHaveLength(1);
    });
});