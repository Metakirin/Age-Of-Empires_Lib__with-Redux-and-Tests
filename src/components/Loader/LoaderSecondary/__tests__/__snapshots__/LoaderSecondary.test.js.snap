// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly "LoaderSecondary" to Match Snapshot 1`] = `
<div
  className="small_loader"
>
  <img
    alt="loading..."
    src="Loading.gif"
  />
</div>
`;
